README.md

```diff
@@ -33,13 +33,19 @@ Now register new extension by adding this code into your `config.neon`:
 extensions:
     slimApi: BrandEmbassy\Slim\DI\SlimApiExtension # Register extension
 
-slimApi: # Configure it
-    slimConfiguration:
-        settings:
-            removeDefaultHandlers: true # It's recommended to disable original error handling 
-                                        # and use your own error handlers suited for needs of your app. 
+services:
+    - BrandEmbassy\Slim\RouterFactory("%configuration%", ...)::create # Register a router factory
```

Comment:
@Zemistr it would be nice if this service was registered automatically by extension, so there is no extra step needed :-)

Reply:
Hmmm... Yeah... probably I can do that... 😉